README.md

```diff
@@ -582,6 +582,7 @@ Libraries to help manage database schemas and migrations.
 ## Filtering and Validation
 *Libraries for filtering and validating data.*
 
+* [HTMLPurifier](http://htmlpurifier.org/) - A HTML filtering library.
```

Comment:
@bocharsky-bw Not sure about this one. The GH doesn't provide any information, apart for the site url, which is where everything is needed to begin with the lib.